src/libraries/System.IO.FileSystem/tests/File/Copy.cs

```diff
@@ -418,5 +419,92 @@ public void EnsureThrowWhenCopyToNonSharedFile()
             using var stream = new FileStream(file1, FileMode.Open, FileAccess.Read, FileShare.None);
             Assert.Throws<IOException>(() => File.Copy(file2, file1, overwrite: true));
         }
+
```

Comment:
We would ideally have a test to ensure it actually does a block copy operation on ReFS. Does anyone know what APIs you can call to check this on Windows?

Reply:
You'd have to ask someone like @erikmav ? We don't in general have special knowledge of ReFS.

Reply:
e.g. on macOS, there is `fcntl` with `F_LOG2PHYS` or `F_LOG2PHYS_EXT`, which can be used to check this.

Reply:
I've been poking around looking for this as e.g. a mode under `fsutil`. Still looking or will ask around.